cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -64,7 +64,7 @@ rollback_etcd3_minor_version() {
   echo "Starting etcd version ${START_VERSION} to capture rollback snapshot."
   if ! start_etcd; then
     echo "Unable to automatically downgrade etcd: starting etcd version ${START_VERSION} to capture rollback snapshot failed."
-    echo "See https://coreos.com/etcd/docs/3.1.11/op-guide/recovery.html for manual downgrade options."
+    echo "See https://coreos.com/etcd/docs/3.1.12/op-guide/recovery.html for manual downgrade options."
```

Comment:
This is a broken link

Reply:
Ah, this doc was only introduced in 3.3.x.  I'll switch to the `latest` ref for this doc link. It's content's really isn't version sensitive so that should get developers to the most up-to-date info.